cmake/Modules/FindLuaJIT.cmake

```diff
@@ -9,7 +9,7 @@
 FIND_PATH(LUA_INCLUDE_DIR luajit.h
 	HINTS
 	$ENV{LUA_DIR}
-	PATH_SUFFIXES include/luajit-2.0 include/luajit-5_1-2.0 include
+	PATH_SUFFIXES include/luajit-2.1 include/luajit-5_1-2.1 include/luajit-2.0 include/luajit-5_1-2.0 include
```

Comment:
why not group both of the `include/luajit-5_1*` together?

Reply:
because I decided to group the versions together. Does it matter?

Reply:
Not really no

Reply:
Did it anyway, don't mind either way